test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Helpers/RunSettingsUtilitiesTests.cs

```diff
@@ -180,5 +180,44 @@ public void GetTestRunParametersIgnoresMalformedKeyValues()
             Assert.IsNotNull(trp);
             Assert.AreEqual(0, trp.Count);
         }
+
+        [TestMethod]
+        public void IsDesignModeReturnsTrueOnNullRunSettings()
```

Comment:
What is the spec when user provides an incorrect runsettings e.g. `<DesignMode>0x1</DesignMode>`?